documentation/IDTA-01001/modules/ROOT/pages/annex/concepts-aas.adoc

```diff
@@ -2,20 +2,26 @@
 Copyright (c) 2023 Industrial Digital Twin Association
 
 This work is licensed under a [Creative Commons Attribution 4.0 International License](
-https://creativecommons.org/licenses/by/4.0/). 
+https://creativecommons.org/licenses/by/4.0/).
 
 SPDX-License-Identifier: CC-BY-4.0
 
 Illustrations:
-Plattform Industrie 4.0; Anna Salari, Publik. Agentur für Kommunikation GmbH, designed by Publik. Agentur für Kommunikation GmbH
+Plattform Industrie 4.0; Anna Salari, Publik.
+Agentur für Kommunikation GmbH, designed by Publik.
+Agentur für Kommunikation GmbH
```

Comment:
See above

Reply:
Fixed in https://github.com/admin-shell-io/aas-specs/pull/452/commits/eb2845590da58d2a5ee077874855d437c0f39299